js/src/components/paid-ads/validateCampaign.js

```diff
@@ -23,6 +24,27 @@ const validateCampaign = ( values ) => {
 		);
 	}
 
+	if (
+		Number.isFinite( values?.amount ) &&
+		Number.isFinite( opts?.budget?.daily_budget )
+	) {
+		const { amount } = values;
+		const { budget, budgetMin } = opts;
```

Comment:
@dsawardekar Please check my latest comment from here: https://github.com/woocommerce/google-listings-and-ads/pull/2583/files#r1756563465
Looks like we can avoid passing the object and just pass the value for `dailyBudget` directly and we don't need to destructure anything here.